SOFTENG310-G5/BrainSomething
# Feat/logo

## Context



This PR updates the branding of the application by changing the browser tab's title and favicon/logo. The previous title and favicon were using the default "React App" configurations, which are now replaced with custom branding elements to align with our BrainSomething's identity.


## What Changed?



- Changed the title in public/index.html to reflect the new branding: "Brain Something".
- Updated the Logo image replacing the default React to our BrainSomething Logo

## How To Review


Manual Testing:
Start the development server (npm start).
Open the application in the browser and verify the following:
The browser tab title displays the updated title (Brain Something).
The favicon in the browser tab shows the new custom logo (brain-logo.png).

## Testing



NA

## Risks


NA

## Notes
NA
